src/modules/handler-executor.js

```diff
@@ -12,8 +14,14 @@ module.exports = (options) => {
 
   return {
     execute: () => new Promise((resolve) => {
+      const cassetteFile = path.join(options.cassetteFolder, options.cassetteFile);
+      const hasCassette = fs.existsSync(cassetteFile);
+      // eslint-disable-next-line no-underscore-dangle
+      const pendingMocks = hasCassette ? nock.load(cassetteFile).map(e => e.interceptors[0]._key) : [];
+      const outOfOrderErrors = [];
+
       consoleRecorder.start();
-      nockBack.setMode('record');
+      nockBack.setMode(hasCassette ? 'lockdown' : 'record');
```

Comment:
Is this not what record mode means? Or was this a bug before?

Reply:
It does. Just making it explicit. No functionality change